src/Persistence/Sql/Oracle/PlatformTrait.php

```diff
@@ -104,17 +104,9 @@ public function getCreateAutoincrementSql($name, $table, $start = 1)
     public function getListDatabasesSQL(): string
     {
         // ignore Oracle maintained schemas, improve tests performance
-        return 'SELECT username FROM sys.all_users'
-            . ' WHERE oracle_maintained = \'N\'';
-    }
-
-    public function getListTablesSQL(): string
```

Comment:
related with https://github.com/doctrine/dbal/issues/5698 and https://github.com/doctrine/dbal/issues/5764